tests/client/amo/components/TestAddonMoreInfo.js

```diff
@@ -108,7 +108,23 @@ describe('<AddonMoreInfo />', () => {
       'Read the privacy policy for this add-on');
     // TODO: Change this to an internal `<Link>` tag and use `assert.equal`
     // once https://github.com/mozilla/addons-frontend/issues/1828 is fixed.
-    assert.include(root.privacyPolicyLink.href,
+    assert.include(root.privacyPolicyLink.props.href,
       '/addon/chill-out/privacy/');
   });
+
+  it('renders the EULA and link', () => {
+    const root = render();
+
+    assert.equal(findDOMNode(root.eulaLink).textContent,
+      'Read the license agreement for this add-on');
+    assert.include(root.eulaLink.props.href, '/addon/chill-out/eula/');
+  });
+
+  it('does not render a EULA if none exists', () => {
+    const partialAddon = deepcopy(fakeAddon);
+    partialAddon.has_eula = false;
```

Comment:
True, this is just copied from another similar test for privacy policy. But I'll tidy it up.